documents-storage-jcr/src/test/java/org/exoplatform/documents/storage/jcr/JCRDocumentFileStorageTest.java

```diff
@@ -176,7 +176,10 @@ public void shareDocument() throws Exception {
     when(currentNode.getPrimaryNodeType()).thenReturn(nodeType);
     when(nodeType.getName()).thenReturn("nt:file");
     when(((ExtendedNode) currentNode).getIdentifier()).thenReturn("123");
+    AccessControlEntry accessControlEntry = new AccessControlEntry("username", PermissionType.READ);
     when(identity.getRemoteId()).thenReturn("username");
+    AccessControlList acl1 = new AccessControlList("username", Arrays.asList(accessControlEntry));
+    when(((ExtendedNode) currentNode).getACL()).thenReturn(acl1);
```

Comment:
Could you add more checks to make sure the permission are well considered ? i.e when we share with read-only and when we share with read-write

Reply:
Done !